broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -167,6 +167,15 @@ class ServiceFabrikApiController extends FabrikBaseController {
   }
 
   getInfo(req, res) {
+    if (!config.enable_swarm_manager) {
+      return res.status(CONST.HTTP_STATUS_CODE.OK)
```

Comment:
Would suggest not have code duplication. Better initialise 
allDockerImagesRetrieved = true 
and execute following code 
```
return docker
       .getMissingImages()
       .then(missingImages => allDockerImagesRetrieved = _.isEmpty(missingImages))
```
only if(config.enable_swarm_manager)==> true or not undefined. Then subsequent code remains same.

Reply:
My idea was to avoid making call to docker client => `getMissingImages()` if swarm is not available.

Reply:
to avoid if 
```
if (!config.enable_swarm_manager)
```
right ? So, I was suggesting the same.

Reply:
Yes agree with Saurav . I think the value of "ready" is a bit misleading , in the sense it should be based on multiple other things in the future. 

So for eg: 
ready = allDockerImagesRetrieved && allBOSHRealsesUploaded ... etc. 

Hence it will be fine to initialise allDockerImagesRetrieved to true and avoid code duplication and just have 1 return statement.

Reply:
@sauravmndl  what if docker.getMissingImages() errors out would not this result in sending the status as true even when there are some images to be downloaded.

Reply:
so in the catch we may have to reset the status